ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -128,25 +133,25 @@
   /**
    * The name of the associated cluster for this node.
    */
-  @Config("clustermap.cluster.name")
+  @Config(CLUSTERMAP_CLUSTER_NAME)
   public final String clusterMapClusterName;
 
   /**
    * The name of the associated datacenter for this node.
    */
-  @Config("clustermap.datacenter.name")
+  @Config(CLUSTERMAP_DATACENTER_NAME)
   public final String clusterMapDatacenterName;
 
   /**
    * The host name associated with this node.
    */
-  @Config("clustermap.host.name")
+  @Config(CLUSTERMAP_HOST_NAME)
   public final String clusterMapHostName;
 
   /**
    * The port number associated with this node.
    */
-  @Config("clustermap.port")
+  @Config(CLUSTERMAP_PORT)
   @Default("null")
```

Comment:
Maybe we can either remove Default from here, or change it to empty string.

Reply:
I don't know why it's there, but I'd prefer not to touch it right now.